components/CircleChart.js

```diff
@@ -1,55 +1,29 @@
 import React from "react";
+import { useAppContext } from '../context/app-context';
 import { Doughnut } from "react-chartjs-2";
-import styles from "../styles/CircleChart.module.scss";
+
+import styles from "../styles/dashboard.module.scss";
 
 export default function CircleChart({ data }) {
+  const { activityColors } = useAppContext();
+
   const chartLabels = data.map((p) => Object.keys(p));
   const chartData = data.map((p) => Object.values(p));
   const config = {
-    labels: [...chartLabels],
+    labels: chartLabels,
     datasets: [
       {
-        data: [...chartData],
-        backgroundColor: [
-          "#FF6384",
-          "#36A2EB",
-          "#FFCE56",
-          "#1dea49",
-          "#ef1aae",
-          "#0b04fa",
-          "#9a1aa0",
-          "#3f218c",
-          "#ebfc05",
-          "#42e6d0",
-          "#bb0935",
-          "#5d9b35",
-          // '#ffa500',
-          // '#ff2500'
-        ],
-        hoverBackgroundColor: [
-          "#FF6384",
-          "#36A2EB",
-          "#FFCE56",
-          "#1dea49",
-          "#ef1aae",
-          "#0b04fa",
-          "#9a1aa0",
-          "#3f218c",
-          "#ebfc05",
-          "#42e6d0",
-          "#bb0935",
-          "#5d9b35",
-          // '#ffa500',
-          // '#ff2500'
-        ],
+        data: chartData,
+        backgroundColor: activityColors,
```

Comment:
I did this on my current branch but a little differently, but I like your way so we should keep this commit.